### PR TITLE
doc: update documentation website with missing features and fixes

### PR DESCRIPTION
**Pull request by phiz71**
**Description**

In this PR we (@leleueri @Okhelifi and I) cherry-picked the commits from the `develop` branch that should already be published. It mainly includes documentation for 3.14.0 features and some fixes.

Another PR like this one will probably be created after the 3.15 release to update the website. After that we will have to change the way we manage our documentation in git.

_We could have merged `develop` in `master`, but due to some delays in products releases, the `develop` branch was in advance regarding the feature released._

### CODE REVIEW

**Review by ytvnr — APPROVED** [review]

**Review by Okhelifi — APPROVED** [review]
LGTM for cockpit part 🚀

**Review by leleueri — APPROVED** [review]

**Comment by phiz71** [comment]
I've asked someone about some "revert commits", to know if we should take them too or not.
I will wait for the answer before merging this PR.

**Pull request merged**
